sites/skeleton.dev/src/routes/(inner)/docs/themes/+page.svelte

```diff
@@ -213,7 +213,7 @@ body { background: red; }
 				</p>
 			</div>
 			<div>
-				<a class="btn variant-filled-secondary font-bold" href="https://csshero.org/mesher/" target="_blank" rel="noreferrer">
+				<a class="btn variant-filled font-bold" href="https://csshero.org/mesher/" target="_blank" rel="noreferrer">
```

Comment:
Was this change intentional or did it slip in from some previous experiments?

Reply:
Intended. I copied the style and fixed it as I went.